src/bootstrap/bootstrap_test.py

```diff
@@ -73,13 +73,21 @@ def test_same_dates(self):
             rustc_stamp.write("2017-06-15None")
         self.assertFalse(self.build.program_out_of_date(self.rustc_stamp_path, self.key))
 
+class ParseToml(unittest.TestCase):
+    def test_parse_quotes(self):
+        rb = bootstrap.RustBuild()
+        rb.config_toml = "[build]\ncc = 'xyz'\n[target.'x86_64-apple-darwin']\ncc = '/not/a/real/compiler'"
+        self.assertEqual(rb.get_toml("cc", "build"), "xyz")
+        cc = rb.get_toml("cc", "target.x86_64-apple-darwin")
+        self.assertEqual(cc, "/not/a/real/compiler")
```

Comment:
This test is going to fail now, since you haven't changed `get_toml`. It's ok to remove it since we're working around the problem in a different way.